### DISCUSSION TITLE
Need help in creating platform independent cp310 version of pycryptodome 3.19.0 package file

### DISCUSSION THREAD

**Discussion by Anjana-MJ** [Legrandin/pycryptodome#788] [Q&A]
I have been attempting to generate a platform-independent ('none' wheel) for pycryptodome version 3.19.0, specifically designed for Python version cp310. The desired file name format should resemble 'pycryptodome-3.19.0-cp310-none-any.whl'.

However, despite my attempts to build this wheel file, I'm encountering an issue where the generated file is named 'pycryptodome-3.19.0-cp35-abi3-win_amd64.whl' instead of containing the expected cp310 ABI tag(I'm using python 3.10.9 virtual env).

Could someone kindly assist me in successfully creating the cp310 'none' wheel file?

Note: 
Attached the output log of python setup.py bdist_wheel --universal command.
[pycrytodome310.odt](https://github.com/Legrandin/pycryptodome/files/13712690/pycrytodome310.odt)

Thanks,
AJ

**Comment by Varbin** [comment]
A universal wheel cannot exist, as PyCryptodome contains – platform specific – compiled C extensions.

**Reply by Anjana-MJ** [reply]
Hi Varbin,
I appreciate your response. I'm wondering if it's possible to generate a wheel file for Python version cp310 instead of cp35. Despite using Python 3.10.9, it consistently producing a cp35 wheel file. Could you please shed some light on the reason behind this?

**Comment by Legrandin** [comment]
Extensions are built against the CPython 3.5 [limited API](https://docs.python.org/3/c-api/stable.html#limited-c-api), so that wheels can be used by all versions of CPython, starting from 3.5.

**Reply by Anjana-MJ** [reply]
Thanks Legrandin
